### DISCUSSION TITLE
articles/life-is-a-good-thing-2022-review

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#178] [Comments]
# articles/life-is-a-good-thing-2022-review

修行

https://hutusi.com/articles/life-is-a-good-thing-2022-review

**Comment by c4710n** [comment]
虽然落在尘网中，但我们可以试着做一只鸟，有那么一刻可以放弃做人，在天空中飞一飞。愿你新年顺利！

**Reply by hutusi** [reply]
哈哈，人是自由的，又无时无刻不在枷锁中。新年快乐！

**Comment by wangyunzi** [comment]
曾经我也处在焦虑不安之中，总感觉自己在浪费时间然后人生无所事事，现在已经解脱了，无所谓了

**Reply by hutusi** [reply]
人生苦短，应该对自己好一点。我现在还谈不上解脱，只能说放下了一些东西吧~

**Comment by leon-zym** [comment]
2022的我也是一度焦虑地影响到身体健康。处在人生的某个节点上，似乎一步追不上节奏就会步步追不上。可能还是自己的修行不够，难以放过自己，与自己和解。

**Reply by hutusi** [reply]
嗯，与自己和解，对自己好一点~

**Comment by conge** [comment]
似乎，浪费了也没什么……

**Reply by hutusi** [reply]
确实。有「浪费」的感觉还是因为焦虑吧，担心失去……

**Comment by Caterpillar945** [comment]
看到大佬也有同款焦虑我就放心了……哈哈

**Reply by hutusi** [reply]
哈哈，现代社会人人都会焦虑呀~~

**Comment by depers** [comment]
感谢分享，祝新的一年顺利！

**Reply by hutusi** [reply]
谢谢，也祝你新年顺利！

**Comment by Natumsol** [comment]
同感，2022一整年都很焦虑，对疫情、对角色的转变（刚成为奶爸）、对工作等等，一度十分痛苦。对我而言，焦虑的副作用则让生活更具痛感，思考问题也更深入，也算是有利也有弊吧。

不过这种痛苦在我读完《洞见：从科学到哲学，打开人类的认知真相》这本书后有所缓解，算是在某种意义上，和焦虑达成了和解吧：「焦虑」的本质是一种「幻觉」，是对环境的应激，它其实并不真实。通过冥想（将自己想象旁观者，俯视、品味这种感觉本身）便可以将这种「幻觉」进行剥离，从而得到缓解。

**Comment by samosxin** [comment]
怎么能沉下心看看书呢，不太喜欢看书，可是又渴望得到一段共鸣。我也不知道我是不是焦虑，想的比做的多吧，做又做不下去的那种烦躁放弃。

**Comment by pimgeek** [comment]
也许这首歌值得试听一下 😃

[小缪 - 浪费你浪费我 ](https://music.163.com/#/song?id=1958990641)

![img](https://p1.music.126.net/2rCuop_WYPVvTcq6COVrqw==/109951167951183525.jpg?param=130y130)

PS：我是你的网站友链中 **思圆笔记** 的博主，最近迁移到新域名了 https://pimgeek.com ，还搭建了一个 [TiddlyWiki 数字花园站](https://pimgeek.com/notes/)，欢迎来访 😎

**Reply by hutusi** [reply]
这首歌很治愈呀。

链接已更新，数字花园很赞：）

**Comment by xieyicn** [comment]
啊啊啊

**Comment by semyin** [comment]
人生更多的是无奈

**Comment by xzLitas** [comment]
现在个人站点都可以开发评论了？

**Comment by jeffwudev** [comment]
怎么过都是过，所以，还是开开心心的过吧，烦恼是身外之物。
